src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/BookingHistory.kt

```diff
@@ -0,0 +1,87 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+/**
+ * This entity represents the history of a booking at a point in time, and is created/saved
+ * whenever a video link booking is CREATED, AMENDED or CANCELLED. Upon each the details of the
+ * related appointments should also be recorded as a list.
+ */
+
+@Entity
+@Table(name = "booking_history")
+class BookingHistory(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val bookingHistoryId: Long = 0L,
+
+  val videoBookingId: Long,
```

Comment:
Just checking you mean to use ID and not type for VideoBooking?

Reply:
Yes. Most definitely - don't want the link to the booking, other than holding the id in history.

Reply:
I realise it would be unidirectional, but just don't want them linked and navigable at all.